test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestProfile.java

```diff
@@ -59,6 +59,16 @@ default boolean disableGlobalTestResources() {
         return false;
     }
 
+    /**
+     * The tags this profile is associated with.
+     * When the {@code quarkus.test.profile.tags} System property is set (its value is a comma separated list of strings)
+     * then Quarkus will only execute tests that are annotated with a {@code @TestProfile} that has at least one of the
+     * supplied (via the aforementioned system property) tags.
+     */
+    default Set<String> tags() {
```

Comment:
I wonder if this should maybe be an annotation instead, i.e. just support @Tag/@Tags on the profile class?

Reply:
I was thinking of that as well.

The reason I didn't do it is because we don't actually tie into the JUnit 5 tag system so having that annotation show up might lead users to believe that if they set the tags in IntelliJ's runner for example, they will get the same behavior as the system property.

I am really on the fence about this, but if you think it will be more clear for users, I'll add it

Reply:
Thats a good point, if we integrated with the existing tags system I would be all for it, but lets leave it as is.